nepalingo-web/src/hooks/useQuotes.tsx

```diff
@@ -1,22 +1,28 @@
 import { useState, useEffect } from "react";
 
-const Languages = ["newari", "tajpuriya", "maithili"] as const;
+interface Quote {
+  text: string;
+  translation: {
```

Comment:
Instead of setting all of the translated languages here, you can `useLanguage` to get the specific language it needs to get translated to